src/modules/TextOverlay/info.json

```diff
@@ -40,6 +40,15 @@
             "type" : "integer",
             "desc": "Enter the font size in pixels.",
             "default": "12"
+        },
+        "custom": {
+            "type": "select",
+            "desc": "Fill Coordinates On Click.",
```

Comment:
This is very cool! Could we change this to a button with a crosshairs icon, maybe? What do you think...

https://fontawesome.com/v4.7.0/icon/crosshairs

`<i class="fa fa-crosshairs"></i>`

Reply:
Sure ,that will be a better choice

Reply:
@jywarren please have a look at these new changes